trunk/SUAVE/Components/Wings/Control_Surfaces/Elevator.py

```diff
@@ -0,0 +1,55 @@
+# Elevator.py
+#
+# Created:  Jan 2020, M. Clarke
+
+# ----------------------------------------------------------------------
+#  Imports
+# ----------------------------------------------------------------------
+
+# SUAVE imports
+from SUAVE.Components.Wings.Control_Surfaces.Control_Surface import Control_Surface  
+
+# ----------------------------------------------------------------------
+#  Attribute
+# ----------------------------------------------------------------------
+
+class Elevator(Control_Surface):
+    """This class is used to define slats in SUAVE
+
+    Assumptions:
+    None
+
+    Source:
+    N/A
+
+    Inputs:
+    None
+
+    Outputs:
+    None
+
+    Properties Used:
+    N/A
+    """ 
+    def __defaults__(self):
+        """This sets the default for slats in SUAVE.
+    
+        Assumptions:
+        None
+
+        Source:
+        N/A
+
+        Inputs:
+        None
+
+        Outputs:
+        None
+
+        Properties Used:
+        N/A
+        """ 
+        
+        self.function = 'elevator'
```

Comment:
tag

Reply:
I have added a tag attribute but will keep the function attribute. Reason: A wing can have multiple control surfaces of the same function

Reply:
I changed "function" to tag and not use the class itself to determine the type of control surface